client/src/components/calendarMonth.jsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import $ from 'jquery';
+import CalendarDay from './calendarDay.jsx';
+import { DivMonth } from '../styles/styles.jsx';
+import { Wrap } from '../styles/styles.jsx';
+import { HeadWrap } from '../styles/styles.jsx';
+import { Span } from '../styles/styles.jsx';
+
+
+class CalendarMonth extends React.Component {
+  
+  constructor(props) {
+    super(props);
+    this.limit = 42;
+  }
+
+  render() {
+    return (
+      <Wrap>  
+        <HeadWrap>
+          <Span>Sun</Span>
+          <Span>Mon</Span>
+          <Span>Tues</Span>
+          <Span>Weds</Span>
+          <Span>Thurs</Span>
+          <Span>Fri</Span>
+          <Span>Sat</Span>
+        </HeadWrap>
+        <DivMonth>
+          { Array.from({ length: this.limit }, (v, i) => i).map((item, i) => {
```

Comment:
I don't know what this is doing with the array.from. I think you can simplify this logic.